pkg/controller/datadogagentdeployment/agent_test.go

```diff
@@ -35,6 +37,10 @@ func Test_newExtendedDaemonSetFromInstance(t *testing.T) {
 			Name:      "confd",
 			MountPath: "/conf.d",
 		},
+		{
+			Name:      "checksd",
+			MountPath: "/checks.d",
```

Comment:
we might want to add `ReadOnly:  true,` here - same for `confd`
( https://github.com/helm/charts/blob/master/stable/datadog/templates/container-agents.yaml#L170 )